core/src/main/scala/kafka/utils/ReplicationUtils.scala

```diff
@@ -27,7 +27,7 @@ import scala.collection._
 
 object ReplicationUtils extends Logging {
 
-  val IsrChangeNotificationPrefix = "isr_change_"
+  private val IsrChangeNotificationPrefix = "isr_change_"
 
```

Comment:
Yes, I did that in another patch. I can put it in this patch to save a separate commit :)